src/optimizer/query_graph.h

```diff
@@ -374,6 +374,7 @@ struct qo_node
   PT_HINT_ENUM hint;		/* hint comment contained in given */
   bool sargable;		/* whether sargs are applicable to this node */
   bool sort_limit_candidate;	/* whether this node is a candidate for a SORT_LIMIT plan */
+  BITSET right_dep_set;		/* outer join dependency for right outer join; to preserve join sequence */
```

Comment:
it's better **right_dep_set** to move below the field "BITSET outer_dep_set"

Reply:
Okay.